lint/verilator_waiver.vlt

```diff
@@ -0,0 +1,69 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// Lint waivers for Verilator
+// See https://www.veripool.org/projects/verilator/wiki/Manual-verilator#CONFIGURATION-FILES
+// for documentation.
+//
+// Important: This file must included *before* any other Verilog file is read.
+// Otherwise, only global waivers are applied, but not file-specific waivers.
+
+`verilator_config
+lint_off -msg PINCONNECTEMPTY
+
+// Filename 'ibex_register_file_ff' does not match MODULE name: ibex_register_file
+// ibex_register_file_ff and ibex_register_file_latch provide two
+// implementation choices for the same module.
+lint_off -msg DECLFILENAME -file "*/rtl/ibex_register_file_ff.sv"
+lint_off -msg DECLFILENAME -file "*/rtl/ibex_register_file_latch.sv"
+
+// Bits of signal are not used: mstatus_n[1:0]
```

Comment:
I think this is no longer needed. `mstatus_n` is renamed to `mstatus_d` and now fully used.